cmake/configs/default.cmake

```diff
@@ -100,8 +100,8 @@ hunter_config(OpenAL VERSION 1.18.2)
 hunter_config(OpenBLAS VERSION 0.2.20-p0)
 hunter_config(OpenCL VERSION 2.1-p3)
 hunter_config(OpenCL-cpp VERSION 2.0.10-p0)
-hunter_config(OpenCV VERSION 3.3.1-p1)
-hunter_config(OpenCV-Extra VERSION 3.3.1)
+hunter_config(OpenCV VERSION 3.4.0-p0)
```

Comment:
Doesn't belong here, please rebase.

Reply:
I am unsure how to do this properly.  Can I delete and recreate the branch on this PR, or should we close it and I resubmit?

Reply:
>  Can I delete and recreate the branch on this PR

Actually I don't know, probably deleting the branch will close pull request. I'm not sure :)

> should we close it and I resubmit?

Push with force should work.